src/main/resources/static/api.yaml

```diff
@@ -1385,6 +1396,65 @@ paths:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
 
+  /patients/recovery/verifyotp:
+    post:
+      tags:
+        - account
+      summary: Verify OTP of the user for reset-password user flow
+      description: Verifies OTP for the given user as part of reset-password flow
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/VerifyOtpRequest'
+          application/xml:
+            schema:
+              $ref: '#/components/schemas/VerifyOtpRequest'
+      responses:
+        '200':
```

Comment:
What would the responsebody be? How do you inform the patient of the recovered CM ID? Are you supposed to send over SMS or display over screen?

Reply:
the response body is there which is sending cm id